docs/recipes/precompiling-with-webpack.md

```diff
@@ -0,0 +1,29 @@
+## Precompiling source files with webpack
+
+The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) mentions precompiling your imported modules as an alternative to runtime compilation, but it doesn't explain how. This recipe shows how to do this using webpack.
+
+###### webpack.config.js
+
+```js
+const nodeExternals = require('webpack-node-externals');
+
+module.exports = {
+	entry: ['src/tests.js'],
+	target: 'node',
+	output: {
+		path: '_build',
+		filename: 'tests.js'
```

Comment:
The `entry: ['src/tests.js']` can be a list of `t.test` or, better yet, `require` or `import` statements pulling in the tests you want to run, because you probably don't want all your tests in one file, unless your project is tiny. As the name suggests, it's the entry point that Webpack will look at to determine what to compile.

The `output.filename` will contain the compiled code.